package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "asciidoctor-vscode",
 	"displayName": "%displayName%",
 	"description": "%description%",
-	"version": "2.9.5",
+	"version": "3.0.0",
```

Comment:
Could you please change it to `3.0.0-dev`? Otherwise `npm version` will fail (unless we explicitly specify `--allow-same-version`)

Reply:
Hopefully, VS Code allows using `3.0.0-dev` in development?

Reply:
```suggestion
	"version": "3.0.0-dev",
```